.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+<!--
+Thanks for contributing to MongoDB Atlas CLI Plugin Kubernetes!
+
+Before you submit your pull request, please review our contribution guidelines:
+https://github.com/mongodb/atlas-cli-plugin-kubernetes/blob/master/CONTRIBUTING.md
+
+Please fill out the information below to help speed up the review process
+and getting you pull request merged!
+-->
+
+## Proposed changes
+
+<!-- 
+Describe the big picture of your changes here and communicate why we should accept this pull request.
+If it fixes a bug or resolves a feature request, be sure to link to that issue. 
+-->
+
+_Jira ticket:_ CLOUDP-#
+
+<!--
+What Atlas CLI Plugin Kubernetes issue does this PR address? (for example, #1234), remove this section if none.
+-->
+
+Closes #[issue number]
+
+## Checklist
+
+<!--
+Check the boxes that apply. If you're unsure about any of them, don't hesitate to ask!
+We're here to help! This is simply a reminder of what we are going to look for before merging your code.
+-->
+
+- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/atlas-cli-plugin-kubernetes/blob/master/CONTRIBUTING.md) (if appropriate)
+- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
+- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
+- [ ] I have run `make fmt` and formatted my code
```

Comment:
[q] do we have `make fmt`? if not we can delete, if yes feel free to resolve!

Reply:
[AtlasCLI](https://github.com/mongodb/mongodb-atlas-cli/blob/master/Makefile#L74) has `make fmt`. May be a nice inclusion considering it should work by copying over the script. WDYT?

Reply:
SGTM!